### PR TITLE
TW-85 schema validation page refresh

### PR DESCRIPTION
**Pull request by jkonrath-postman**

### CODE REVIEW

File: src/pages/docs/designing-and-developing-your-api/validating-elements-against-schema.md [hunk 1]
```diff
@@ -38,92 +39,82 @@ You can validate your API schema in Postman, and validate your elements (documen
...
+If there is an issue with your schema JSON or YAML syntax, you will see a warning. Look for errors indicated in the editor and hover over them for more detail. Postman will only be able to validate your schema elements if the JSON or YAML is itself well-formed.
 
 If there are no errors, Postman will indicate in the lower pane that your schema is valid.
 
 ## Validating elements
 
 > Element validation is available for OpenAPI 3.0 and WSDL 1.1 and 2.0 schemas.
 
-After you make changes to your API schema, you can validate associated elements to determine if they will also need changes to stay in sync. For example, if you add a new method to your API schema, validation will indicate you also need to add the method to your tests.
+You can compare your API schemas against documentation, mock servers, tests, and monitors associated with it to determine if the elements need changes to stay in sync. For example, if you add a new method to your API schema, validation will indicate you need to add the method to your associated documentation. Or when you add a new element to an API, you can run a validation to check the element against the current API schema for issues.
```
Comment (HeatherStenson) [inline src/pages/docs/designing-and-developing-your-api/validating-elements-against-schema.md:60]:
```suggestion
You can compare your API schema against documentation, mock servers, tests, and monitors associated with it to determine if the elements need changes to stay in sync. For example, if you add a new method to your API schema, validation will indicate you need to add the method to your associated documentation. Or when you add a new element to an API, you can run a validation to check the element against the current API schema for issues.
```

File: src/pages/docs/designing-and-developing-your-api/validating-elements-against-schema.md [hunk 2]
```diff
@@ -38,92 +39,82 @@ You can validate your API schema in Postman, and validate your elements (documen
...
-* API elements - this can be a [mock server](/docs/designing-and-developing-your-api/developing-an-api/#adding-a-mock-server), [documentation](/docs/designing-and-developing-your-api/developing-an-api/#adding-documentation), [tests](/docs/designing-and-developing-your-api/testing-an-api/), or a [monitor](/docs/designing-and-developing-your-api/observing-an-api/)
+To validate a new element, select **Validate** in the validate column next to the element. To re-validate an element, select the status, then select **Validate Again**.
 
-> Once the validation is complete, a status will appear next to the element indicating a checkmark if no issues have been found, or a warning message stating `Issues found`.
-
-When you add a new element to an API, you need to trigger a validation to check it against the current API schema. The results of the validation will appear next to the element.
+Once the validation is complete, a status will appear next to the element indicating a checkmark if no issues have been found, or a warning message stating that issues are found. If there are issues, select the status, then select **Review Issues** to review them.
```
Comment (HeatherStenson) [inline src/pages/docs/designing-and-developing-your-api/validating-elements-against-schema.md:66]:
```suggestion
Once the validation is complete, a checkmark will appear next to the element if no issues have been found, or a warning message stating that issues are found. If there are issues, select the status, then select **Review Issues** to review them.
```

File: src/pages/docs/designing-and-developing-your-api/validating-elements-against-schema.md [hunk 3]
```diff
@@ -38,92 +39,82 @@ You can validate your API schema in Postman, and validate your elements (documen
...
 ## Accessing issues
 
 You can review the issues found during validation to fix issues in your collections or schemas.
 
-If an issue arises when you run a request, click the warning message next to the name of the request (e.g. **1 issue**). This will open a side-panel on the right indicating detailed information about which component of the request is affected and what the issue itself is:
+If an issue arises when you run a request, select the warning message next to the name of the request (for example, **1 issue**). This will open a panel on the right indicating detailed information about which component of the request is affected and what the issue itself is:
 
-* A direct link to the API against which the request is validated—click the link to open the API within the Postman app.
+* A direct link to the API against which the request is validated. Select the link to open the API within Postman.
 * More details on where the issue lies within the request.
```
Comment (HeatherStenson) [inline src/pages/docs/designing-and-developing-your-api/validating-elements-against-schema.md:87]: I know you didn't write this section, but this list doesn't make a ton of sense - I think maybe it's just a continuation of the previous idea of what the "detailed information" contains. Maybe nix the list and subsume it into the section on line 84?

**Review by HeatherStenson — APPROVED** [review]
Looks good! A few notes, no blockers.

**Review by mark-anthony-postman — APPROVED** [review]
LGTM! Heather already caught the few minor things I saw 🙂

File: src/pages/docs/designing-and-developing-your-api/validating-elements-against-schema.md (hunk 1 above)
Comment (jkonrath-postman) [reply src/pages/docs/designing-and-developing-your-api/validating-elements-against-schema.md:60]: There can be one or more schemas per API, so it should be plural.

**Pull request merged**